auth0_flutter/lib/src/web/js_interop_utils.dart

```diff
@@ -31,4 +31,22 @@ class JsInteropUtils {
 
     return obj;
   }
+
+  // TODO: replace with `dartify` from `dart:js_interop_unsafe` when migrating to WASM
```

Comment:
I added these here, to make the migration to WASM easier. Then we only need to change these two, instead of all callsites